### PR TITLE
Add FCOS MobileNetV2 backbone

### PR DESCRIPTION
**Pull request by lhcalibur**
image
I can provide model checkpoint file.

### CODE REVIEW

**Comment by ggjy** [comment]
@lhcalibur Hi, I reimplement the MobileNetV2 on both one-stage and two-stage detector, the training loss seems normal, but the evalutaion (distributed training) is 0. The log is as follows, do you meet situation like this? Thanks.

Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
2019-08-02 19:25:31,634 - INFO - Epoch [1][3665/3665]	lr: 0.02000, bbox_mAP: 0.0000, bbox_mAP_50: 0.0000, bbox_mAP_75: 0.0000, bbox_mAP_s: 0.0000, bbox_mAP_m: 0.0000, bbox_mAP_l: 0.0000, bbox_mAP_copypaste: 0.000 0.000 0.000 0.000 0.000 0.000
2019-08-02 19:25:44,849 - INFO - Epoch [2][50/3665]	lr: 0.02000, eta: 5:55:16, time: 0.262, data_time: 0.051, memory: 6130, loss_cls: 2.1071, loss_bbox: 0.8964, loss_centerness: 0.6620, loss: 3.6655
2019-08-02 19:25:55,860 - INFO - Epoch [2][100/3665]	lr: 0.02000, eta: 5:54:28, time: 0.222, data_time: 0.012, memory: 6130, loss_cls: 2.3207, loss_bbox: 0.8720, loss_centerness: 0.6620, loss: 3.8547
2019-08-02 19:26:06,866 - INFO - Epoch [2][150/3665]	lr: 0.02000, eta: 5:53:39, time: 0.220, data_time: 0.010, memory: 6130, loss_cls: 2.1023, loss_bbox: 0.8658, loss_centerness: 0.6638, loss: 3.6320
2019-08-02 19:26:17,728 - INFO - Epoch [2][200/3665]	lr: 0.02000, eta: 5:52:49, time: 0.218, data_time: 0.011, memory: 6130, loss_cls: 2.5522, loss_bbox: 0.8750, loss_centerness: 0.6629, loss: 4.0902
2019-08-02 19:26:28,650 - INFO - Epoch [2][250/3665]	lr: 0.02000, eta: 5:52:00, time: 0.218, data_time: 0.011, memory: 6130, loss_cls: 2.0592, loss_bbox: 0.8717, loss_centerness: 0.6634, loss: 3.5943

**Comment by BlueBlueFF** [comment]
where is the pretrained model of backbone?

**Comment by BlueBlueFF** [comment]
> image
> I can provide model checkpoint file.

Can you share model checkpoint file, thanks~~

**Comment by huihuixiaohuihui** [comment]
hello. I want to know the model's fps

**Comment by lhcalibur** [comment]
> hello. I want to know the model's fps
You can refer to https://github.com/tianzhi0549/FCOS
image

**Comment by lhcalibur** [comment]
> > image
> > I can provide model checkpoint file.
> 
> Can you share model checkpoint file, thanks~~

链接：https://pan.baidu.com/s/12stqEW5r6CpiIN-gRCB6Sw 
提取码：gwet 
复制这段内容后打开百度网盘手机App，操作更方便哦

**Comment by BlueBlueFF** [comment]
> > > image
> > > I can provide model checkpoint file.
> > 
> > 
> > Can you share model checkpoint file, thanks~~
> 
> 链接：https://pan.baidu.com/s/12stqEW5r6CpiIN-gRCB6Sw
> 提取码：gwet
> 复制这段内容后打开百度网盘手机App，操作更方便哦

I use this checkpoint and only got 29.5 ap on coco, do you use other test tricks like multi-scale test or something?

**Comment by hajungong007** [comment]
image
我使用你的代码训练出来的效果和你的效果相差有点大，请问你是使用了其他trick吗？我使用了4张卡，每张卡４张图训练。

**Comment by ZwwWayne** [comment]
Close this PR as MobileNetv2 has been supported in https://github.com/open-mmlab/mmdetection/pull/5122

**Comment by yang-yk** [comment]
@lhcalibur  hello, can you provide the checkpoint and the corresponding config file? The download link failed. Many thanks!!!
